src/plugins/AB-Why3/preludes/dune

```diff
@@ -8,5 +8,11 @@
 )
 
 (cram
+ (package alt-ergo)
```

Comment:
This should be part of package `alt-ergo-plugin-ab-why3, not `alt-ergo`

Reply:
Oh I didn't know `alt-ergo-plugin-ab-why3` depends on the binary also :)

Reply:
Yeah, plugins have to depend on `alt-ergo` because they install stuff to the `alt-ergo` sites.

Reply:
Yep, I saw it. It's sad we cannot share dune sites between several packages.

Reply:
What do you mean? The site mechanism is what we use to share data across several packages, it's just that we want the site to be called "alt-ergo", not "alt-ergo-lib" (which should be merged back into alt-ergo anyways)